pom.xml

```diff
@@ -25,7 +25,7 @@
   </parent>
 
   <artifactId>async-http-client</artifactId>
```

Comment:
<artifactId>async-http-client2</artifactId> as version 2 is not binary compatible